src/components/Contentful/Prose.js

```diff
@@ -2,23 +2,75 @@ import React from 'react'
 import { Prose } from '@madetech/frontend'
 
 export default function ContentfulProse({
-  columnWidth,
-  columnOffset,
+  customClasses,
+  extraLargeColumnWidth,
+  extraLargeColumnOffset,
+  extraSmallColumnWidth,
+  extraSmallColumnOffset,
   html,
   id,
+  largeColumnWidth,
+  largeColumnOffset,
+  mediumColumnWidth,
+  mediumColumnOffset,
+  smallColumnWidth,
+  smallColumnOffset,
   textAlign,
 }) {
-  let className = 'contentful-prose'
-  if (textAlign) className += ` text-${textAlign}`
+  let proseClassName = 'contentful-prose'
+  if (textAlign) proseClassName += ` text-${textAlign}`
```

Comment:
Possibly just wrap the body of this if in curly braces as well? Can keep it on one line though